index.html

```diff
@@ -151,22 +162,28 @@ <h2>Food, Friends, and Fun</h2>
             <div class="image_card3"></div>
             <div>
                 <h1>Event Schedule</h1>
-                <table class="schedule">
-                    <tr><td>09:00</td><td>Check-In Begins</td></tr>
-                    <tr><td>09:00</td><td>Free Breakfast </td></tr>
-                    <tr><td>09:30</td><td>Opening Ceremony</td></tr>
+                <h3>Friday April 17th</h3>
+                <table class="schedule" name="2020-4-17">
+                    <tr><td style="width: 60px">09:30</td><td>Opening Ceremony (join via <a href="https://washington.zoom.us/j/798388880">Zoom</a>)</td></tr>
                     <!-- Use HTML entity for table column spacing -->
-                    <tr><td>10:00 &nbsp; &nbsp;</td><td>Team Formation Mixer, hosted by ACM-W</td></tr>
+                    <tr><td>10:00</td><td>Team Formation Mixer, hosted by ACM-W</td></tr>
                     <tr><td>10:00</td><td>Hacking Begins</td></tr>
-                    <tr><td>10:45</td><td>Talk: Introduction To Cloud Services</td></tr>
-                    <tr><td>12:30</td><td>Free Lunch</td></tr>
-                    <tr><td>13:15</td><td>Lightning Talks: Tales from the Cloud</td></tr>
+                    <tr><td>10:00</td><td>Mentors go online (join via <a href="https://discord.gg/EDBbxmt">Discord</a>)</td</tr>
+                    <tr><td>10:45</td><td>Talk: Introduction To Cloud Services (join via <a href="https://washington.zoom.us/j/951323956">Zoom</a>)</td></tr>
+                    <tr><td>13:15</td><td>Lightning Talks: Tales from the Cloud (join via <a href="https://washington.zoom.us/j/573514052">Zoom</a>)</td></tr>
```

Comment:
not too familiar with zoom, are these supposed to be different links?

Reply:
Yes, I set them up as different meetings. I suppose they _could_ all be the same meeting. Will think about that.